package.json

```diff
@@ -52,31 +54,40 @@
   },
   "dependencies": {
     "@google-cloud/common": "^0.16.0",
+    "@types/express": "^4.11.0",
+    "@types/hapi": "^16.1.11",
+    "@types/request": "^2.0.9",
+    "@types/restify": "^5.0.6",
```

Comment:
Why do these `@types/...` packages have to be dependencies vs devDependencies?

Reply:
They should be in devDependencies.

Reply:
Fixed